api/PositionOptions.json

```diff
@@ -159,14 +154,20 @@
               },
               {
                 "version_added": "10",
-                "version_removed": "15"
```

Comment:
This is a substantive change that seems out of scope with the topic of the PR. Can you submit this change separately with the story behind it?

Reply:
Opera for Android began using Blink in [version 14](https://dev.opera.com/blog/opera-14-for-android-is-out/).